### DISCUSSION TITLE
NiceDCV integration

### DISCUSSION THREAD

**Discussion by crabby345** [amazonlinux/amazon-linux-2022#43] [Ideas]
Can a nod be given to the NiceDCV team when considering incorporation of Gnome/KDE into AL2022. The scaling above 1080p on AL2 (non-GPU) leaves a lot to be desired and it seems to be related to desktop limitations/interactions.

Sounds like there's an opportunity to get this right as desktop still needs to be integrated Gnome 40/41 etc.

Appreciate this could easily be badged as someone else's problem, but these things need to interact from both sides.

**Comment by stewartsmith** [comment]
We are engaged with the NiceDCV team, and are conscious of the challenges of running modern desktop environments in the cloud.

**Comment by stewartsmith** [comment]
Since NiceDCV is not Open Source software, we are not considering including it in Amazon Linux. If that changes, we can reconsider including it.
